common/changes/miwhea-fabric-core-6_2017-02-07-22-38.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Update to Fabric Core 6.0.0",
+      "type": "patch"
```

Comment:
It still might cause a lot of problems, especially if a repo consuming fabric is, for whatever reason, still using the old version of Fabric Core. That could lead to css collisions and other issues similar to the ones we saw with the button change.

I could also just be overly cautious now though.